tuf/api/metadata.py

```diff
@@ -653,30 +652,63 @@ class VerificationResult:
     """Signature verification result for delegated role metadata.
 
     Attributes:
-        verified: True, if threshold of signatures is met.
-        signed: Set of delegated keyids, which validly signed.
-        unsigned: Set of delegated keyids, which did not validly sign.
+        threshold: Number of required signatures.
+        signed: dict of keyid:Key containing the keys that have signed.
+        unsigned: dict of keyid:Key containing the keys that have not signed.
```

Comment:
nit: Would you mind to change this?

```suggestion
        signed: dict of keyids to Keys that have signed.
        unsigned: dict of keyids to Keys that have not signed.
```

The second colon throws off the parser in my head. 😆